adrs/5541831634/README.md

```diff
@@ -0,0 +1,157 @@
+# React project files and directories names conversion
+
+* **State:** Draft
+* **Created:** 2020-12-01
+* **Tags:** react
+
+## Context
+
+Across the ecosystem, we can find all sorts of files and directory naming
+conventions in React projects.
+
+The variety of conventions causes misalignment and shifts focus away from more
+important matters.
+
+Although there are no silver bullets for this topic, committing to a convention
+will reduce cognitive load by triggering pattern recognition, enabling engineers
+to focus their brainpower on their tasks.
```

Comment:
I agree with this!  In my lifetime Prettier has been the most effective org/tool making this point.  They spell it out in their intro docs: https://prettier.io/docs/en/why-prettier.html#building-and-enforcing-a-style-guide

> By far the biggest reason for adopting Prettier is to stop all the on-going debates over styles. It is generally accepted that having a common style guide is valuable for a project and team but getting there is a very painful and unrewarding process. People get very emotional around particular ways of writing code and nobody likes spending time writing and receiving nits.

Reply:
It'd be really dope if we had tooling like Prettier or ESLint to help automate and enforce a file structure style guide.

Reply:
I will be working in some CLI that would help us as well